### ISSUE TITLE
💻 Removal idea: Quiz

### ISSUE THREAD

**Issue by Felienne** [hedyorg/hedy#5989]
As with Parsons, it is not really well integrated into a trajectory and can be better done on paper. But, I could be swayed here if quizzes are used a lot, can you check @jpelay?

**Comment by MarleenGilsing** [comment]
@Felienne I think we can agree to remove the quizzes, or integrate them with the slides or workbooks. So @boryanagoncharenko can work on deleting the current quizzes?

**Comment by Felienne** [comment]
Yes, I think the workbooks is a much better place for them, so we can remove them. The content can be merged into the the workbooks (maybe we should make a separate issue for that?

**Comment by boryanagoncharenko** [comment]
As discussed with @MarleenGilsing, I created a sub-issue to remove the Weblate component.

**Comment by boryanagoncharenko** [comment]
I forgot to put the check for minimum quiz score behind the flag
